package.json

```diff
@@ -40,7 +42,6 @@
     "@storybook/addon-links": "^7.0.0-rc.6",
     "@storybook/addon-styling": "^0.3.2",
     "@storybook/blocks": "^7.0.0-rc.6",
-    "@storybook/react": "7.0.0-rc.6",
```

Comment:
@musllim you should add this package ```"@storybook/react"``` again in the devdependencies and remove it from the dependencies